types/express-rate-limit/index.d.ts

```diff
@@ -43,12 +43,6 @@ declare namespace rateLimit {
         resetAll(): void;
     }
 
-    interface Message {
-        status: number;
-        message: string;
-        [key: string]: any;
-    }
```

Comment:
Mark them optional, this will fix accepting anything and still offer common props hints in IDEs.
```ts
interface Message {
    status?: number;
    message?: string;
    [key: string]: unknown;
}
```
and would be nice to have exact use-case with error prop added to tests